Website update following v1.2.0

adriguir opened the pull request:
Website update following v1.2.0. @gautierbureau I have updated all the links, even if Windows and MacOS versions are not yet released but this should be temporary or do we have blocking points?

Possible improvements in the long-term could include:

- Adding the logo on the right part on large screens

- Use versions of the logo that don't have a white background (especially for DynaWaltz and DynaWave specific pages). 

If someone wants to do it, please feel free.

adriguir commented:
@bongrainmat @marcochiaramello @rosiereflo @cossartque @RomainLosseauRTE fyi

gautierbureau commented:
I made a small improvement for the links between to go directly to said news instead of just the whole page

adriguir commented:
> I made a small improvement for the news links to go directly to said news instead of just the whole page

That's a nice idea, thanks! I just corrected the links and changed the news order to be the same in the page and in the banner
